main/src/cgeo/geocaching/maps/mapsforge/MapsforgeMapSource.java

```diff
@@ -8,12 +8,15 @@
 
 import org.mapsforge.map.android.graphics.AndroidGraphicFactory;
 import org.mapsforge.map.layer.cache.TileCache;
+import org.mapsforge.map.layer.download.tilesource.AbstractTileSource;
 import org.mapsforge.map.layer.download.tilesource.OpenStreetMapMapnik;
 import org.mapsforge.map.model.MapViewPosition;
 import org.mapsforge.v3.android.maps.mapgenerator.MapGeneratorInternal;
 
 public class MapsforgeMapSource extends AbstractMapSource {
 
+    public static final String MAPNIK_TILE_DOWNLOAD_UA = "cgeo";
```

Comment:
Wasn't there a more complete User Agent agreed to in other issues/PRs? See cgeo#6611

Reply:
`cgeo` is not a valid user agent. Please read this for details: https://stackoverflow.com/a/2601492

Reply:
If I read the RFC (https://tools.ietf.org/html/rfc7231) correctly is 'cgeo' a valid UA, while the addition of a version could be considered. Even though it would be the only real one to use if the UA string would have been used as it was originally intended, I do not suggest to replace it globally, but just for tile downloads from OSM (if the continue to let us do it in the first place).

Reply:
Why not use the UA SammysHP suggested in cgeo#6611? Are there technical reasons against it?

> What do you think about
> 
> ```
> Mozilla/5.0 (Linux; Android <android-release>) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.0.0 Safari/537.36 cgeo/<cgeo-release>
> ```

Reply:
It is IMHO the final target to use such kind of UA throughout the app.
But as this kind of change needs some testing, we change to this simple UA for mapsforge loading only first and later implement a common solution.
At least this would be my proposal.

Reply:
> Why not use the UA SammysHP suggested in #6611? Are there technical reasons against it?
There is no hard reason against it, besides wasting resources on transmission (tiny) and UA parsing on OSM side (likely tiny as well), but there are no reasons for using it in this context either. Did you check the discussing WRT UA-based blocking and the 'Tile Usage Policy' on OSM side (in the underlying issue #7647)?

Reply:
Ok, then this just needs to be tested, correct?

@SammysHP doesn't the SO link you posted also say the `cgeo` string is ok? the version seems to be optional, at least that is my reading of the EBNF ( [ ... ] denoting optional)?

@rsudev Sorry I didn't read it (and the policy) in detail before now. Yes, it seems that `cgeo` is the easiest solution, however the other points in the policy are dubiously fulfilled by c:geo as well (resource usage, attribution), and using the more complete UA is IMHO not faking another app's UA, if the `cgeo` string is included.

So I vote for provisionally accepting your PR (if I have any voting rights at all, being a newbie and all 😅)

The other points are addressed in the issue already aren't they...

Reply:
@jakedot Yeah, you're right. It might be valid with respect to the RFC but far away from the from the usual `Mozilla/5.0`.

Reply:
tested it, seems to work fine for me